packages/dask-task-models-library/src/dask_task_models_library/container_tasks/events.py

```diff
@@ -47,15 +47,15 @@ class Config(BaseTaskEvent.Config):
 
 class TaskLogEvent(BaseTaskEvent):
     log: str
-    log_level: int = logging.INFO
+    log_level: int
```

Comment:
MINOR: I think we could make a small improvemnt by having a literal named e.g. ``LogLevelInt``
```python
CRITICAL = 50
FATAL = CRITICAL
ERROR = 40
WARNING = 30
WARN = WARNING
INFO = 20
DEBUG = 10
NOTSET = 0
```

Reply:
why the hell did they do not use an enum in python logging?